src/diff/props.js

```diff
@@ -11,12 +11,17 @@ import options from '../options';
  */
 export function diffProps(dom, newProps, oldProps, isSvg) {
 	for (let i in newProps) {
-		if (i!=='children' && i!=='key' && (!oldProps || oldProps[i]!=newProps[i])) {
+		if (i!=='children' && i!=='key' && i!=='value' && (!oldProps || oldProps[i]!=newProps[i])) {
 			setProperty(dom, i, newProps[i], oldProps[i], isSvg);
 		}
 	}
+
+	// `value` needs to be applied after an input's `min`, `max` and `step`
+	// attribute are set. See preact/#850
```

Comment:
I wonder if there are any other dependencies here? I presume there is one between `type` and `value`, which is also covered by this change.

Reply:
Not sure but we didn't receive any bug reports for these so far, even during the 8.x release line. So I guess browsers deal with `type` themselves.